app.js

```diff
@@ -3,6 +3,18 @@ import { renderDogCard } from './render-utils.js';
 
 const dogListContainer = document.getElementById('dog-list-container');
 
+async function fetchDogs() {
```

Comment:
Great work! Here we grab one dog out of the array and use our render dog card to render that one dog and append all of the dogs to our container.